apps/testrunner/actions/phantomas_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+import phantomas
+
+from . import Action
+from phantomas.results import Runs
+
+
+logger = logging.getLogger(__name__)
+
+
+class PhantomasTest(Action):
+    REQUIRED_PARAMS = (
+        'url',
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(PhantomasTest, self).__init__(*args, **kwargs)
+
+    def run(self):
+        try:
+            phantomas_runner = phantomas.Phantomas(
+                url=self.params['url'],
+                runs=self.params['retries']
+            )
+            phantomas_result = phantomas_runner.run()
+            if hasattr(phantomas_result,'_runs'):
```

Comment:
Public API of Results does not support getting "offenders" for *all* metrics and it simplifies collecting results a lot. Thus if we already rely upon some implementation details I thought that would be the easiest way. What do you think?